scripts/history-update-check.sh

```diff
@@ -6,11 +6,10 @@
 #   $ history-update-check.sh
 #
 # This should be run from the root of the repository.
-set -e
-
-! git diff --exit-code master -- HISTORY.md > /dev/null
-
-if [ $? ]; then
+res=$(git diff origin/master --name-only | grep ^HISTORY.md | wc -l)
+echo "Files Changed:"
+git diff --name-only
```

Comment:
Is this to just show what changed in files with the CI?

Reply:
Yes, since it's its own test case, I figured it could have some more output. :)